compiler/src/dmd/iasm.d

```diff
@@ -13,9 +13,15 @@
 
 module dmd.iasm;
 
+import core.stdc.stdio;
```

Comment:
Unused import

Reply:
I use it for debug printfs. It doesn't cost anything to leave it in.